### PR TITLE
Readme: say that the repo is unmaintained

### PR DESCRIPTION
**Pull request by Luap99**
We focus on netavark and aardvark-dns.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # dnsname plugin
 
+## IMPORTANT
+
+This repository is no longer actively maintained. Our new development efforts are happening in [netavark](https://github.com/containers/netavark/) and [aardvark-dns](https://github.com/containers/aardvark-dns/) that should provide a better alternative.
```
Comment (edsantiago) [inline README.md:5]: Any time my brain sees the word "new", it yells at me.

Sorry to be a nag, but would you mind adding "As of 2023"? E.g.
>As of 2023, this repository is ...

**Comment by edsantiago** [comment]
/lgtm

Thank you

**Pull request merged**